task_base.py

```diff
@@ -8,6 +8,14 @@
 
 PROJECTS = "projects"
 
+class EETaskError(Exception):
+    def __init__(self, ee_statuses, *args, **kwargs):
```

Comment:
Shouldn't the arg be `ee_messages`?

Reply:
I named it that because of it's the result of `getTaskStatus`